package.json

```diff
@@ -34,7 +34,7 @@
     "@octokit/webhooks": "13.2.7",
     "@sinclair/typebox": "0.32.23",
     "@supabase/supabase-js": "2.42.0",
-    "@ubiquibot/permit-generation": "1.3.1",
+    "@ubiquibot/permit-generation": "1.6.0",
```

Comment:
```suggestion
    "@ubiquibot/permit-generation": "*",
```

Reply:
I would use a particular version. Otherwise it's harder to debug permits and find out where exactly (`conversation-rewards` plugin or `permit-generation` package) things got wrong. @gentlementlegen What do you think?

Reply:
We simply can log it. 

Originally, I would append meta data with the revision, in the comment. This feature should be added back again.

For now, I know that we can trace back the get of actions run so just log the revision there, if the GitHub actions UI doesn't already show this information. 

Upon yarn install it might show it though?

Reply:
Updated https://github.com/ubiquibot/conversation-rewards/pull/108/commits/5315bfc08684088d36b850ce6c9bd0b942c27a63